### ISSUE TITLE
Non-case endpoints don't work

### ISSUE THREAD

**Issue by ChefAndy** [harvard-lil/capstone#521]
500s for any endpoint other than the case endpoint.

**Comment by bensteinberg** [comment]
I haven't checked this locally, but I'm guessing it's almost certainly a deployment error... What tier are you looking at?

**Comment by ChefAndy** [comment]
Nothing that can't wait until you get back, @bensteinberg. 

https://api.capapi.org/v1/courts/

**Comment by bensteinberg** [comment]
On stage, the only top-level API routes that don't work are volumes and reporters.

**Comment by bensteinberg** [comment]
OK, here's the error in question:
```
    [pid: 29448|app: 0|req: 27/94] 162.158.62.226 () {58 vars in 1022 bytes} [Fri Sep 28 13:13:26 2018] GET /v1/volumes/ => generated 4594 by
tes in 80 msecs (HTTP/1.1 500) 6 headers in 207 bytes (1 switches on core 0)
    Got existing for query-count:02719b4d858737768e0edf5f3417c0dc
    2018-09-28 13:13:30,090 ERROR module=exception, process_id=29447, Internal Server Error: /v1/reporters/
    Traceback (most recent call last):
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/django/core/handlers/exception.py", line 35, in inner
        response = get_response(request)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/django/core/handlers/base.py", line 128, in _get_response
        response = self.process_exception_by_middleware(e, request)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/django/core/handlers/base.py", line 126, in _get_response
        response = wrapped_callback(request, *callback_args, **callback_kwargs)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
        return view_func(*args, **kwargs)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/rest_framework/viewsets.py", line 103, in view
        return self.dispatch(request, *args, **kwargs)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
        response = self.handle_exception(exc)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/rest_framework/views.py", line 443, in handle_exception
        self.raise_uncaught_exception(exc)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
        response = handler(request, *args, **kwargs)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/rest_framework/mixins.py", line 42, in list
        page = self.paginate_queryset(queryset)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/rest_framework/generics.py", line 173, in paginate_queryset
        return self.paginator.paginate_queryset(queryset, self.request, view=self)
      File "./capapi/pagination.py", line 22, in paginate_queryset
        return super().paginate_queryset(queryset, *args, **kwargs)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/rest_framework/pagination.py", line 542, in paginate_queryset
        following_position = self._get_position_from_instance(results[-1], self.ordering)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/rest_framework/pagination.py", line 772, in _get_position_from_instance
        field_name = ordering[0].lstrip('-')
    IndexError: tuple index out of range
```

**Comment by bensteinberg** [comment]
(Not a deployment error, I think.)

**Comment by bensteinberg** [comment]
After #528, on stage I'm getting
```
    [pid: 31945|app: 0|req: 6/24] 162.158.62.202 () {62 vars in 2924 bytes} [Fri Sep 28 14:29:41 2018] GET /v1/citations/ => generated 4594 b
ytes in 13 msecs (HTTP/1.1 500) 6 headers in 207 bytes (1 switches on core 0)
    Got existing for query-count:0528b23179f9b2c817f04f27682a27a0
    2018-09-28 14:29:43,266 ERROR module=exception, process_id=31941, Internal Server Error: /v1/citations/
    Traceback (most recent call last):
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/django/core/handlers/exception.py", line 35, in inner
        response = get_response(request)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/django/core/handlers/base.py", line 128, in _get_response
        response = self.process_exception_by_middleware(e, request)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/django/core/handlers/base.py", line 126, in _get_response
        response = wrapped_callback(request, *callback_args, **callback_kwargs)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
        return view_func(*args, **kwargs)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/rest_framework/viewsets.py", line 103, in view
        return self.dispatch(request, *args, **kwargs)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
        response = self.handle_exception(exc)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/rest_framework/views.py", line 443, in handle_exception
        self.raise_uncaught_exception(exc)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
        response = handler(request, *args, **kwargs)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/rest_framework/mixins.py", line 42, in list
        page = self.paginate_queryset(queryset)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/rest_framework/generics.py", line 173, in paginate_queryset
        return self.paginator.paginate_queryset(queryset, self.request, view=self)
      File "./capapi/pagination.py", line 22, in paginate_queryset
        return super().paginate_queryset(queryset, *args, **kwargs)
      File "/usr/local/share/.virtualenvs/capstone/lib/python3.5/site-packages/rest_framework/pagination.py", line 505, in paginate_queryset
        self.ordering = self.get_ordering(request, queryset, view)
      File "./capapi/pagination.py", line 29, in get_ordering
        'Invalid ordering. Expected string or tuple, but got %s' % (ordering,)
    AssertionError: Invalid ordering. Expected string or tuple, but got []
```

**Comment by jcushman** [comment]
Oops, I somehow thought citations wasn't a real endpoint. Sent another PR.

**Comment by bensteinberg** [comment]
closed in #529
